app/views/api/messages/_message.json.jbuilder

```diff
@@ -0,0 +1,10 @@
+json.id message.id
+json.from_user_id message.from_user_id
+json.to_user_id message.to_user_id
```

Comment:
Do we want to include the user names as well?

Reply:
User names are not available in message itself, I would have to either modify the model and include those two, or query for user names in order to return them.

They would be nice to have, but they would cost some database lookups

Reply:
The question is how likely is it that the user of the API will just come back and make user details calls to get the names anyway? If they're going to do that then it would be better to do it here where we can use joins to get the names more efficiently...